### ISSUE TITLE
Bug: 6 tests fail with Firefox

### ISSUE THREAD

**Issue by lobax** [mozilla/metrics-graphics#814]
Using firefox 57.0.1 under OSX, 6 tests fail when running `grub test`. The same failures appear after an upgrade to 58.0.1, and have been replicated with a different machine running Windows 10. The tests do not fail with Chrome.

**Comment by wlach** [comment]
Yep, I can reproduce this as well with Firefox 60 (nightly). Needs investigation.
